app/models/miq_queue.rb

```diff
@@ -109,6 +109,7 @@ def self.put(options)
     options[:queue_name]  ||= create_with_options[:queue_name] || "generic"
     options[:msg_timeout] ||= create_with_options[:msg_timeout] || TIMEOUT
     options[:task_id]      = $_miq_worker_current_msg.try(:task_id) unless options.key?(:task_id)
+    options[:tracking_label] = Thread.current["tracking_label"] || options[:task_id] unless options.key?(:tracking_label)
```

Comment:
@bzwei huh. turns out both will do the same thing.
I can change it, but had already merged this for the vmdb-logger.

It looks like you can use symbols and strings interchangeably.

```ruby
>Thread.current[:test] = "abc"
>puts Thread.current["test"]
abc
>Thread.current["test"] = "cde"
>puts Thread.current[:test]
cde
```